Makefile.am

```diff
@@ -1,4 +1,4 @@
-SUBDIRS = fpmsyncd neighsyncd intfsyncd portsyncd orchagent swssconfig
+SUBDIRS = fpmsyncd neighsyncd intfsyncd portsyncd orchagent swssconfig vlanconfd
```

Comment:
i prefer vlanmgr than vlanconfd

Reply:
There is already bgpconfd.  We may change to vlanmgr from vlanconfd, then it introduces name inconsistency.

As suggested by @qiluo-msft in https://github.com/Azure/sonic-swss/pull/357,  probably we may merge the files of vlan configuration enforcer and intf configuration enforcer into one flat folder while keeping them in separate processes,  other future configuration enforcer (in c++) could follow the same practice. 

I don't have strong opinion about the naming of configuration enforcer folder and configuration enforcers themselves, but to avoid multiple iterations of name change and folder rearrangement, could @lguohan @taoyl-ms @qiluo-msft  @stcheng agree on the following names:

folder name:  cfgagent
vlan configuration enforcer:  vlanconfd
intf configuration enforcer:    intfconfd
 
Or please help give names (and folder structure) you all agree on, thanks!